DTS-STN/Service-Canada-Labs
# SCCH/MSCAD Overview Page

# SCCH/MSCAD Overview Page

This PR covers adding the SCCH/MSCAD page to the site, as well as it's card on the homepage. Includes small changes to styling including adding grid to paragraph and heading elements so that they take up 8/12 grid spaces per Service Canada Design System.

## Test Instructions

1. Go to home page
2. Navigate to My Service Canada Account dashboard through the card and see that it matches [design](https://www.figma.com/file/ke3SrZNGMP3AjwzMMuEgbo/%5Bcurrent%5D-BDM-DECD-SC-Labs?type=design&node-id=14348-33953&mode=design&t=tSaD84K5ib3m4i2n-0)
3. Toggle to french and navigate the same way

## Definition of Done
